test/fakeintake/server/server.go

```diff
@@ -440,10 +452,13 @@ func (fi *Server) handleConfigureOverride(w http.ResponseWriter, req *http.Reque
 		writeHTTPResponse(w, response)
 		return
 	}
+	if payload.Method == "" {
```

Comment:
Should we add something to check that `payload.Method` is in a list of supported method. Because IIUC, here if we set the Method to something that does not exist the call to `safeSetResponseOverride` will crash the fakeintake with a `nil pointer deference`

Reply:
Fair, will return error on invalid method, allowing empty

Reply:
Fixed